swlor_2da/racialtypes.2da

```diff
@@ -153,15 +153,18 @@
-157   Cyborg               Cy      16858054   16858054     16858060          16858066     16858072      IR_CYBORG     10005        0           0           0           0           0           0           20          ****      RACE_FEAT_HUMAN   16858047    1            RACIAL_TYPE_CYBORG               25     4                     1            
-158   Cathar               Ca      16858074   16858074     16858075          16858076     16858077      IR_CATHAR     10006        0           1           -1          -1          0           1           20          ****      RACE_FEAT_HUMAN   16858047    1            RACIAL_TYPE_CATHAR               25     4                     1            
-159   Trandoshan           Tr      16858205   16858205     16858208          16858207     16858206      IR_TRANNY     10007        1           0           -1          -1          0           1           20          ****      RACE_FEAT_HUMAN   16858047    1            RACIAL_TYPE_TRANDOSHAN           25     4                     1            
-160   Mirialan             Mi      16858305   16858305     16858306          16858307     16858308      IR_MIRIALAN   10008        -1          1           0           0           1           -1          20          ****      RACE_FEAT_HUMAN   16858047    1            RACIAL_TYPE_MIRIALAN             25     4                     1            
-161   Echani               Ec      16858309   16858309     16858310          16858311     16858312      IR_ECHANI     10009        0           1           0           -1          -1          1           20          ****      RACE_FEAT_HUMAN   16858047    1            RACIAL_TYPE_ECHANI               25     4                     1            
-162   MonCalamari          Mc      16858313   16858313     16858314          16858315     16858316      IR_MONCAL     10010        0           0           -1          -1          1           1           20          ****      RACE_FEAT_HUMAN   16858047    1            RACIAL_TYPE_MON_CALAMARI         25     4                     1            
-163   Ugnaught             Ug      16858488   16858488     16858489          16858490     16858491      IR_UGNAUGHT   10011        1           -1          0           -1          0           1           20          ****      RACE_FEAT_HUMAN   16858047    1            RACIAL_TYPE_UGNAUGHT             25     4                     1            
+152   Bothan               Bo      16858049   16858049     16858055          16858061     16858067      IR_BOTHAN     10000        -1          1           0           0           0           0           20          ****      RACE_FEAT_HUMAN   16858047    1            RACIAL_TYPE_BOTHAN               25     4                     1            
```

Comment:
These racial stat changes appear to be configured for revamp. Legacy has different stat progression, so we need to revert these changes and introduce equivalent stats for the new races you've added.